prime-router/src/main/kotlin/Report.kt

```diff
@@ -134,8 +134,8 @@ data class ReportStreamFilterResult(
     val filteredTrackingElement: String,
     val filterType: ReportStreamFilterType?,
     val filteredObservationDetails: String? = null,
+    override val scope: ActionLogScope = ActionLogScope.translation,
```

Comment:
Can you explain why this is set to translation? I'm not quite following the reason for this, and it seems wrong? We're not translating anything here....

Edit: Okay I see it was like this before, all you did was move it up to the constructor so in FHIRReceiverFIlter you can pass in `ActionLogScope.report`, which makes more sense in that context.